user_tools/tests/spark_rapids_tools_e2e/resources/scripts/hdfs/common.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export hdfs_dir="/tmp/hadoop"
+export hadoop_version="3.3.6"
```

Comment:
This will cause errors when the default version in the core/pom file changes.
Here we should read a global variable and set the `${hadoop.version}` to be passed to the pom file. If not set then use the default value.